cl/search/documents.py

```diff
@@ -539,3 +547,276 @@ def prepare_person_child(self, instance):
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    caseName = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    case_name_full = fields.TextField(
+        attr="case_name_full",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="case_name_full", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    suitNature = fields.TextField(
+        attr="nature_of_suit",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="nature_of_suit", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    cause = fields.TextField(
+        attr="cause",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(attr="cause", analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    juryDemand = fields.TextField(
+        attr="jury_demand",
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(
+                attr="jury_demand", analyzer="english_exact"
+            ),
+        },
+        search_analyzer="search_analyzer",
+    )
+    jurisdictionType = fields.TextField(
+        attr="jurisdiction_type",
+    )
+    dateArgued = fields.DateField(attr="date_argued")
+    dateFiled = fields.DateField(attr="date_filed")
+    dateTerminated = fields.DateField(attr="date_terminated")
+    dateArgued_text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    dateFiled_text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    dateTerminated_text = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    docket_slug = fields.KeywordField(attr="slug", index=False)
+    assignedTo = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
+    referredTo = fields.TextField(
+        analyzer="text_en_splitting_cl",
+        fields={
+            "exact": fields.TextField(analyzer="english_exact"),
+        },
+        search_analyzer="search_analyzer",
+    )
```

Comment:
We used to have `assigned_to_id` and `referred_to_id`, which I don't see here. It's worth looking at the other solr fields one more time to see if any are missing.

Reply:
Sure, I added these fields and verified we that we have all fields currently indexed in Solr.